src/components/email-flow/EmailFlowProof.tsx

```diff
@@ -25,7 +25,7 @@ export default function ({
     <>
       <HeaderText extraLeading>{headerTitle}</HeaderText>
       <BodyText>{statusText}</BodyText>
-      <ReadyEmailProof proof={proof} />
+      <Proof type="Email" original={proof.original} proof={proof} />
```

Comment:
Hmm, why can't we extract `original` inside the component? You have a `proof` there

Reply:
Proof is optional parameter